x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_scheduling.tsx

```diff
@@ -200,25 +203,51 @@ export const ConnectorSchedulingComponent: React.FC = () => {
         </EuiFlexItem>
         {shouldShowAccessControlSync && (
           <EuiFlexItem>
-            <SchedulePanel
-              title={i18n.translate(
-                'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.title',
-                { defaultMessage: 'Document Level Security' }
-              )}
-              description={i18n.translate(
-                'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.description',
-                {
-                  defaultMessage:
-                    'Control the documents users can access, based on their permissions and roles. Schedule syncs to keep these access controls up to date.',
-                }
+            <EuiFlexGroup direction="column">
+              <EuiFlexItem>
+                <SchedulePanel
+                  title={i18n.translate(
+                    'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.title',
+                    { defaultMessage: 'Document Level Security' }
+                  )}
+                  description={i18n.translate(
+                    'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.description',
+                    {
+                      defaultMessage:
+                        'Control the documents users can access, based on their permissions and roles. Schedule syncs to keep these access controls up to date.',
+                    }
+                  )}
+                >
+                  <ConnectorContentScheduling
+                    type={SyncJobType.ACCESS_CONTROL}
+                    index={index}
+                    hasPlatinumLicense={hasPlatinumLicense}
+                  />
+                </SchedulePanel>
+              </EuiFlexItem>
+              {isDocumentLevelSecurityDisabled && (
+                <EuiFlexItem>
+                  <EuiCallOut
+                    title={i18n.translate(
+                      'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.dlsDisabledCallout.title',
+                      { defaultMessage: 'Permissions and identity syncs not allowed' }
+                    )}
+                    color="warning"
+                    iconType="iInCircle"
+                  >
+                    <p>
+                      {i18n.translate(
+                        'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.dlsDisabledCallout.text',
+                        {
+                          defaultMessage:
+                            'Enable DLS connector’s configurations to activate these options.',
```

Comment:
```suggestion
                            'Enable document level security for this connector to activate these options.',
```

Reply:
Can we link to the configuration tab here?

In that case it would be "[Enable document level security]() for this connector to activate these options."

Reply:
I'll try

Reply:
@efegurkan if that's not worth the investment, we can just reword to:

_Enable document level security for this connector in the configuration tab to activate these options._